cli/src/commands/auth.rs

```diff
@@ -208,6 +208,28 @@ pub async fn handle_auth_revoke_token(
     Ok(ExitCode::Ok)
 }
 
+/// Create a new Locksmith token.
+pub async fn handle_auth_create_token(
+    config: Config,
+    matches: &clap::ArgMatches,
+    timeout: Option<u64>,
+) -> CommandResult {
+    // Create a client with our auth token attached.
+    let api = PhylumApi::new(config, timeout).await?;
+
+    let token_name = matches.get_one::<String>("token-name").unwrap().to_string();
+    let ignore_certs = api.config().ignore_certs();
+    let api_uri = &api.config().connection.uri;
+
+    let token =
+        auth::handle_auth_flow(AuthAction::Login, Some(token_name), ignore_certs, api_uri).await?;
+
+    eprintln!();
+    print_user_success!("API token: {token}");
```

Comment:
Formatting the output like this does not allow for the suggestion in #1190:

> It may also be useful to ensure that the generated token is the only output on stdout when running the command. That would allow use in scripting:
>
> ```sh
> MY_TOKEN=$(phylum auth new-token)
> ```

Was this overlooked or is there a reason the output can't be made this way?